### DISCUSSION TITLE
EVIL SUGGESTION!!!11!

### DISCUSSION THREAD

**Discussion by davwat** [CluckCluckChicken/Magnifier#45] [big chungus]
[https://scratch.mit.edu/discuss/topic/43677](https://scratch.mit.edu/discuss/topic/43677) HOW DARE THEY NOT SEE MAGNIFIER COMING IN THE FUTURE!!!1!!111!! :((((((( 😡😡😡👿👿👿😡👿😡👿😡

**Comment by FunctionalMetatable** [comment]
sussy

**Comment by JackyBorderCollie** [comment]
very sus and evil
